chaos_genius/views/anomaly_data_view.py

```diff
@@ -425,6 +357,46 @@ def kpi_anomaly_retraining(kpi_id):
         return jsonify({"msg": f"retraining failed for KPI: {kpi_id}, KPI id is None"})
 
 
+def _get_dimensions_values(kpi_id):
```

Comment:
Type annotations are missing.
```suggestion
def _get_dimensions_values(kpi_id: int) -> Dict[str, List[str]]:
```

Reply:
Fixed.